src/test/java/org/junit/tests/assertion/MultipleFailureExceptionTest.java

```diff
@@ -67,6 +68,16 @@ public void assertEmptyThrowsMutipleFailureExceptionForManyThrowables() throws E
         }
     }
 
+    @Test
+    public void assertEmptyErrorListConstructorFailure() throws Exception {
+        try {
+            new MultipleFailureException(Collections.<Throwable>emptyList());
+            fail();
+        } catch (Exception expected) {
+            assertThat(expected, instanceOf(IllegalArgumentException.class));
+            assertTrue(expected.getMessage().equals("Argument error list cannot be empty"));
```

Comment:
This won't give a useful message if it fails. You can use Hamcrest to assert that the message contains the expected text